Mathlib/Algebra/Invertible/Defs.lean

```diff
@@ -241,26 +238,27 @@ def Invertible.mul [Monoid α] {a b : α} (_ : Invertible a) (_ : Invertible b)
   invertibleMul _ _
 #align invertible.mul Invertible.mul
 
-theorem mul_right_inj_of_invertible [Monoid α] (c : α) [Invertible c] :
+theorem mul_right_inj_of_invertible [Monoid α] {a b : α} (c : α) [Invertible c] :
     a * c = b * c ↔ a = b :=
   ⟨fun h => by simpa using congr_arg (· * ⅟c) h, congr_arg (· * _)⟩
 
-theorem mul_left_inj_of_invertible [Monoid α] (c : α) [Invertible c] :
+theorem mul_left_inj_of_invertible [Monoid α] {a b : α} (c : α) [Invertible c] :
```

Comment:
Can you put all these lemmas in a section with `variable [Monoid α] {a b : α} (c : α) [Invertible c]`?

Reply:
This makes the diff much bigger; and I don't want anyone to be saying "look how much mess it makes to clear up autoimplicit.

I'll make a follow-up PR.

bors merge